rotkehlchen/history/events/structures/base.py

```diff
@@ -311,7 +311,10 @@ def serialize_for_csv(self, fiat_value: FVal) -> dict[str, Any]:
         right after the 'asset' in the serialized dictionary. Note that
         'fiat_value' is not in USD but in the user-selected currency.
         """
-        new_dict: dict[str, Any] = {'direction': self.maybe_get_direction()}
+        new_dict: dict[str, Any] = {
+            'direction': self.maybe_get_direction(),
+            'asset_symbol': self.asset.symbol_or_name(),
```

Comment:
what's the order here? It should appear in the column right next to the asset id. Does this do it?